frontend/src/app/testResults/uploads/CsvSchemaDocumentation.tsx

```diff
@@ -265,7 +268,7 @@ const CsvSchemaDocumentation = () => {
             </a>{" "}
           </p>
 
-          <h4 className="margin-top-4">Column headers and order</h4>
+          <h3 className="margin-top-4">Column headers and order</h3>
```

Comment:
the "Required, requested, and optional fields" heading looks like it might need to be moved up to `h3` as well - can't commit on the line directly but it's below this

Reply:
fixed